cafebabel/articles/models.py

```diff
@@ -23,10 +23,20 @@ class Article(db.Document, UploadableImageMixin):
     creation_date = db.DateTimeField(default=datetime.datetime.utcnow)
     publication_date = db.DateTimeField()
 
+    _translations = None
+
     meta = {
         'allow_inheritance': True
     }
 
+    @property
+    def translations(self):
+        if not self._translations:
+            from .translations.models import Translation  # NOQA: circular :/
+            self._translations = list(
```

Comment:
1. What about generating a dict `{'lang': article}` once and for all?
2. How do we invalidate that cache in case of newly added translation?